backend-engineers/README.md

```diff
@@ -23,8 +23,8 @@ Write an API (with several endpoints) to:
 ### Technical requirements
 
 - Use a private Github repository
-- Use Scala/ZIO/http4s
```

Comment:
Remove explicit mention to Http4s. In DevTools we're not using it anymore and zio-http is easier to use in a ZIO stack (No need to know `FunctorK`)

Reply:
:crying_cat_face: